src/store/actions/groupActions.js

```diff
@@ -0,0 +1,207 @@
+import { get } from 'lodash/fp'
```

Comment:
I think it's helpful to sequester all of these into one file, but I would call it `groupRelationshipActions.js` vs `groupActions.js` as there are other group related actions with their own files.